### PR TITLE
add "Create Model" setup step

### PR DESCRIPTION
**Pull request by teallarson**
## Change description

> Adds a Setup Step in the UI for creating a Model. 
> **This PR will not be ready to merge until we have a doc to point users to that explains Models.**
> model--01--add
> (screenshot is from updated Config UI docs, hence the red rectangles)

## Related issues

Does this PR fix a Github Issue?

> No

### Development

- [X] Application changes have been tested appropriately

### Impact

- [X] Code follows company security practices and guidelines
- [X] Security impact of change has been considered
- [X] Performance impact of change has been considered
- [X] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

> No impact to those areas

### Code review

- [X] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [X] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: core/src/modules/ops/setupSteps.ts [hunk 1]
```diff
@@ -53,6 +54,19 @@ export namespace SetupStepOps {
         return setting.value !== setting.defaultValue;
       },
     },
+    add_a_model: {
+      key: "add_a_model",
+      title: "Add a Model",
+      description:
+        "Models define the shape of your Records.  Grouparoo uses a Profile Model by default.  You can add new Models and customize them when you begin adding Sources and Properties.",
```
Comment (evantahler) [inline core/src/modules/ops/setupSteps.ts:61]: We may want to tweak this language a bit, but this is 👍 for now!

**Review by evantahler — APPROVED** [review]

File: core/src/modules/ops/setupSteps.ts (hunk 1 above)
Comment (teallarson) [reply core/src/modules/ops/setupSteps.ts:61]: Happy to tweak it if you have a suggestion!

Comment (teallarson) [reply core/src/modules/ops/setupSteps.ts:61]: Should I merge it now?  The docs links go to a 404 currently as the models/records updates haven't been done yet.

**Comment by evantahler** [comment]
It's OK to merge now - we won't release a non-alpha until WWW is updated too.

**Pull request merged**